measure_extinction/utils/make_obsdata_from_model.py

```diff
@@ -311,6 +308,27 @@ def make_obsdata_from_model(
             "%s/Models/%s_full.fits" % (output_path, output_filebase), overwrite=True
         )
 
+    iue_file = "%s_iue.fits" % (output_filebase)
+    if not only_dat:
+        # IUE mock observation
+        # Resolution approximately 300
+        iue_fwhm_pix = rbres / 300.0
+        g = Gaussian1DKernel(stddev=iue_fwhm_pix / 2.355)
+        # Convolve data
+        nflux = convolve(otable["FLUX"].data, g)
+
+        iue_table = QTable()
+        iue_table["WAVELENGTH"] = otable["WAVELENGTH"]
+        iue_table["FLUX"] = nflux * fluxunit
+        iue_table["NPTS"] = otable["NPTS"]
+        iue_table["ERROR"] = Column(np.full((len(iue_table)), 1.0)) * fluxunit
+
+        rb_iue = merge_iue_obsspec([iue_table])
+        rb_iue["SIGMA"] = rb_iue["FLUX"] * 0.0
```

Comment:
Multiplying by zero gives zero uncertainties.  These are models, hence there is not uncertainty (ERROR or SIGMA).  No difference, just words meaning the same thing.  Different observed data uses different words for the uncertainty.

Reply:
Added more comments.

Reply:
Still somewhat confused here. For the iue_table["ERROR"], you set the values to 1, but for the rb_iue["SIGMA"], you set them to 0. Is there a particular reason for that? I guess in the second case, you could just set them to 0 instead of multiplying the flux by 0 (which I assume takes less memory/time).